diff --git a/node_modules/@parcel/transformer-css/lib/CSSTransformer.js b/node_modules/@parcel/transformer-css/lib/CSSTransformer.js
index 562c8ae..a13d1e1 100644
--- a/node_modules/@parcel/transformer-css/lib/CSSTransformer.js
+++ b/node_modules/@parcel/transformer-css/lib/CSSTransformer.js
@@ -110,7 +110,7 @@ var _default = new (_plugin().Transformer)({
             if (cssModulesConfig !== null && cssModulesConfig !== void 0 && cssModulesConfig.dashedIdents && !asset.isSource) {
               cssModulesConfig.dashedIdents = false;
             }
-            cssModules = cssModulesConfig !== null && cssModulesConfig !== void 0 ? cssModulesConfig : true;
+            cssModules = cssModulesConfig !== null && cssModulesConfig !== void 0 ? cssModulesConfig : { pattern: "A[hash]_[local]" };
           }
         }
         res = (0, _lightningcss().transform)({
