diff --git a/node_modules/@parcel/transformer-svg-react/lib/SvgReactTransformer.js b/node_modules/@parcel/transformer-svg-react/lib/SvgReactTransformer.js
index e3d9cfd..17a68f8 100644
--- a/node_modules/@parcel/transformer-svg-react/lib/SvgReactTransformer.js
+++ b/node_modules/@parcel/transformer-svg-react/lib/SvgReactTransformer.js
@@ -57,6 +57,17 @@ function _core() {
 
 function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { default: obj }; }
 
+const svgrConfig = {
+  jsxRuntime: "automatic",
+  expandProps: false,
+  template: (variables, { tpl }) => tpl`
+    function ${variables.componentName} (${variables.props}) {
+      return (${variables.jsx});
+    }
+    ${variables.exports};
+  `,
+}
+
 var _default = new (_plugin().Transformer)({
   async loadConfig({
     config
@@ -93,7 +104,7 @@ var _default = new (_plugin().Transformer)({
     var _config$svgr;
 
     let code = await asset.getCode();
-    const jsx = await (0, _core().transform)(code, {
+    const jsx = await (0, _core().transform)(code, svgrConfig, {
       svgoConfig: config.svgo,
       ...config.svgr,
       runtimeConfig: false
