diff --git a/node_modules/@parcel/packager-html/lib/HTMLPackager.js b/node_modules/@parcel/packager-html/lib/HTMLPackager.js
index 18d7851..ab28b1f 100644
--- a/node_modules/@parcel/packager-html/lib/HTMLPackager.js
+++ b/node_modules/@parcel/packager-html/lib/HTMLPackager.js
@@ -108,7 +108,7 @@ var _default = new (_plugin().Packager)({
     let renderConfig = config === null || config === void 0 ? void 0 : config.render;
     let {
       html
-    } = await (0, _posthtml().default)([tree => insertBundleReferences(referencedBundles, tree), tree => replaceInlineAssetContent(bundleGraph, getInlineBundleContents, tree)]).process(code, { ...renderConfig,
+    } = await (0, _posthtml().default)([tree => replaceInlineAssetContent(bundleGraph, getInlineBundleContents, tree)]).process(code, { ...renderConfig,
       xmlMode: bundle.type === 'xhtml',
       closingSingleTag: bundle.type === 'xhtml' ? 'slash' : undefined
     });
