diff --git a/node_modules/@parcel/optimizer-swc/lib/SwcOptimizer.js b/node_modules/@parcel/optimizer-swc/lib/SwcOptimizer.js
index 88ec762..db3593b 100644
--- a/node_modules/@parcel/optimizer-swc/lib/SwcOptimizer.js
+++ b/node_modules/@parcel/optimizer-swc/lib/SwcOptimizer.js
@@ -83,15 +83,27 @@ var _default = new (_plugin().Optimizer)({
     try {
       result = await (0, _core().transform)(code, {
         jsc: {
-          target: 'es2022',
           minify: {
-            mangle: true,
-            compress: true,
             ...userConfig,
+            mangle: {
+              ie8: true,
+              safari10: true,
+            },
+            compress: {
+              passes: 2,
+              keep_fargs: false,
+              negate_iife: false,
+              side_effects: false,
+              properties: false,
+              ie8: true,
+            },
             toplevel: bundle.env.outputFormat === 'esmodule' || bundle.env.outputFormat === 'commonjs',
             module: bundle.env.outputFormat === 'esmodule'
           }
         },
+        env: {
+          targets: bundle.env.engines.browsers,
+        },
         minify: true,
         sourceMaps: !!bundle.env.sourceMap,
         configFile: false,
@@ -159,7 +171,9 @@ var _default = new (_plugin().Optimizer)({
       }
       let sourcemapReference = await getSourceMapReference(sourceMap);
       if (sourcemapReference) {
-        minifiedContents += `\n//# sourceMappingURL=${sourcemapReference}\n`;
+        let sourceMapDir = bundle.target.publicUrl + _path().default.dirname(bundle.name).replaceAll(_path().default.sep, "/").replace(/^\.\//, '');
+        sourceMapDir += sourceMapDir.endsWith('/') ? '' : '/';
+        minifiedContents += `\n//# sourceMappingURL=${sourceMapDir}${sourcemapReference}\n`;
       }
     }
     return {
