diff --git a/node_modules/@parcel/transformer-svg-react/lib/SvgReactTransformer.js b/node_modules/@parcel/transformer-svg-react/lib/SvgReactTransformer.js
index 017663a..2a9467a 100644
--- a/node_modules/@parcel/transformer-svg-react/lib/SvgReactTransformer.js
+++ b/node_modules/@parcel/transformer-svg-react/lib/SvgReactTransformer.js
@@ -33,6 +33,18 @@ function _core() {
   return data;
 }
 function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { default: obj }; }
+
+const svgrConfig = {
+  jsxRuntime: "automatic",
+  expandProps: false,
+  template: (variables, { tpl }) => tpl`
+    function ${variables.componentName} (${variables.props}) {
+      return (${variables.jsx});
+    }
+    ${variables.exports};
+  `,
+}
+
 var _default = new (_plugin().Transformer)({
   async loadConfig({
     config
@@ -50,7 +62,7 @@ var _default = new (_plugin().Transformer)({
   }) {
     var _config$svgr;
     let code = await asset.getCode();
-    const jsx = await (0, _core().transform)(code, {
+    const jsx = await (0, _core().transform)(code, svgrConfig, {
       svgoConfig: config.svgo,
       ...config.svgr,
       runtimeConfig: false
